### PR TITLE
hammer to cleanup kubernetes calico annotations of offline nodes in CHI@edge

### PR DESCRIPTION
**Pull request by JOUNAIDSoufiane**

### CODE REVIEW

File: hammers/scripts/k8s_calico_ip_cleanup.py [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+        offline_nodes = [
+            node.metadata.name
+            for node in nodes.items
+            if node.status.conditions is None
+            or all(
+                cond.status != "True"
+                for cond in node.status.conditions
+                if cond.type == "Ready"
+            )
+        ]
```
Comment (msherman64) [inline hammers/scripts/k8s_calico_ip_cleanup.py:69]: There are multiple reasons a node may not be `ready`, including that calico is still starting.

we should be more specific, such as:
> `node.kubernetes.io/unreachable`: Node is unreachable from the node controller. This corresponds to the `NodeCondition` `Ready` being `Unknown`.

see https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/#taint-based-evictions

**Review by msherman64 — APPROVED** [review]
Should be good to merge so long as we target a more specific kubernetes condition so we don't accidentally make calico loop.

File: hammers/scripts/k8s_calico_ip_cleanup.py (hunk 1 above)
Comment (JOUNAIDSoufiane) [reply hammers/scripts/k8s_calico_ip_cleanup.py:69]: right, on it

File: hammers/scripts/k8s_calico_ip_cleanup.py
Comment (JOUNAIDSoufiane) [inline hammers/scripts/k8s_calico_ip_cleanup.py:1]: @msherman64 good to merge ?

**Review by msherman64 — APPROVED** [review]
LGTM, thanks!

**Pull request merged**